facebook/reason
# [AtomReason] NuclideReason -> AtomReason

Test plan: 
- change config.cson field to AtomReason
- merge
- make clean
- make
- opam upgrade reason
- (test npm start in AtomReason to see nothing new produced)
- restart editor
- check hyperclick
- autocomplete
- formatting
- type hint
